cmake/generate_pkg.cmake.in

```diff
@@ -18,7 +18,7 @@ set(GINKGO_PROJECT_NAME "@CMAKE_PROJECT_NAME@")
 set(GINKGO_DESCRIPTION "@Ginkgo_DESCRIPTION@")
 set(GINKGO_VERSION "@Ginkgo_VERSION@")
 
-set(GINKGO_INSTALL_PKGCONFIG_DIR "${GINKGO_PKG_INSTALL_PREFIX}/${GINKGO_INSTALL_LIBDIR}/pkgconfig")
+set(GINKGO_INSTALL_PKGCONFIG_DIR "$ENV{DESTDIR}${GINKGO_PKG_INSTALL_PREFIX}/${GINKGO_INSTALL_LIBDIR}/pkgconfig")
```

Comment:
Just to be sure, this works because the install prefix is always an absolute path, and DESTDIR is prepended without `/`?

Reply:
So I tried it with `DESTDIR=/tmp/dummy cmake --install .` and it somewhere automatically added  `/` to `DESTDIR`. If I would put an extra `/` either on the CLI or in the script, there would be an `//`, which of course doesn't really matter. Also, the make documentation describes using it without the `/`: https://www.gnu.org/software/make/manual/html_node/DESTDIR.html

Reply:
I think the `/` comes from the beginning of `/usr/local`, so that should be fine, I just wanted to confirm that the install prefix is always an absolute path.

Reply:
Right, I misunderstood you there a bit. As long as `CMAKE_INSTALL_PREFIX` is absolute it should work. In the case that someone does for example `cmake --install . --prefix relative/dir` then it doesn't seem to work. I will look into that, but I don't think this would have a relevant use case anyway.

Reply:
Ok, now I ensure that `GINKGO_PKG_INSTALL_PREFIX` is absolute, so it works in both cases.